pkg/backporter/backporter.go

```diff
@@ -553,30 +558,118 @@ func CreateCloneHandler(client bugzilla.Client, allTargetVersions sets.String, m
 			handleError(w, err, fmt.Sprintf("unable to fetch bug details- Bug#%d", bugID), http.StatusNotFound, endpoint, bugID, m)
 			return
 		}
-		// Create a clone of the bug
-		cloneID, err := client.CloneBug(bug)
+		if !allTargetVersions.Has(req.FormValue("release")) {
+			handleError(w, fmt.Errorf("invalid argument - %s is not a valid TargetRelease", req.FormValue("release")), fmt.Sprintf("invalid argument - %s is not a valid TargetRelease", req.FormValue("release")), http.StatusBadRequest, endpoint, bugID, m)
```

Comment:
Please also include which target releases are valid :)

Reply:
sorry i missed that, i fixed that now